NNPTPZ1/Fractal.cs

```diff
@@ -0,0 +1,158 @@
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+
+using Mathematics;
+
+namespace App
+{
+    class Fractal
+    {
+        internal readonly int NUMBER_OF_ITERATIONS = 30;
```

Comment:
there is too much fields and in many cases you prefer this mess over direct return values from methods, this mades it really bad for comprehension of whats going on here...